--- sawfish-1.5.0/sawfish.desktop.xdg	2009-07-03 15:46:05.000000000 +0200
+++ sawfish-1.5.0/sawfish.desktop	2009-07-05 05:08:45.000000000 +0200
@@ -1,7 +1,5 @@
 [Desktop Entry]
 Name=Sawfish
-Name[el]=Πριονόψαρο (Sawfish)
-
 Type=Application
 Exec=sawfish
 TryExec=sawfish
@@ -9,10 +7,7 @@
 Terminal=false
 NoDisplay=true
 
-X-GNOME_WMName=Sawfish
+X-GNOME-WMName=Sawfish
 X-GNOME-Autostart-Phase=WindowManager
 X-GNOME-Provides=windowmanager
 X-GNOME-Autostart-Notify=true
-
-[Window Manager]
-SessionManaged=true
